sonar-orchestrator/src/main/java/com/sonar/orchestrator/container/SonarDistribution.java

```diff
@@ -121,11 +123,22 @@ public boolean isRelease() {
     return version.isRelease();
   }
 
+  @Deprecated
   public SonarDistribution activateLicense(String pluginKey) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)